packages/client/src/clientPages/home/api/getRecommend.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+
+import {
+   APIApi,
+   Pageable,
+   PageResponseQuestionPostSimpleResponse,
+} from '@server-api/api'
+import { config } from '@shared/api/config'
+
+export const useRecommendList = (
+   dto: {
+      pageable: Pageable
+   },
+   options?: UseQueryOptions<PageResponseQuestionPostSimpleResponse>,
+) =>
+   useQuery({
```

Comment:
`as`를 사용하여 타입을 단언할 수도 있지만, `useQuery`의 타입 정의를 보면, 인자로 받은 queryFn의 반환 값인 `Promise<T>`를 기반으로 전달받은 반환 값인 data를  `data: T`로 타입을 자동으로 추론해줍니다. 만약 `queryFn`에서 타입이 명시되어 있지 않다면, `useQuery`의 제네릭 인자를 통해 반환 값의 타입을 직접 지정할 수도 있습니다.

```suggestion
   useQuery<PageResponseQuestionPostSimpleResponse>({
```

Reply:
오 그러면 as를 사용할 필요 없이 useQuery의 제네릭 인자로 타입 지정을 해주면 되겠네요!!
좋은 정보 감사합니다 👍🏻